config/routes.rb

```diff
@@ -350,6 +350,7 @@
           as: :in_person_ready_to_verify
       get '/in_person/usps_locations' => 'in_person/usps_locations#index'
       put '/in_person/usps_locations' => 'in_person/usps_locations#update'
+      get '/in_person/addresses' => 'in_person/address_search#index'
```

Comment:
a *light* convention (not 100% followed) we have is that we put our API endpoints under the `/api` namespace. Since this controller returns JSON and isn't expected to return HTML, I'd consider moving this there. But that could also happen in a separate PR when we actually have UI that makes s request here

Reply:
> also, because this is PII, going to change to a post.

Good catch!

It also has me wondering if it'd make sense to change the specs to use [Mx Fakerson's address](https://github.com/18F/identity-idp/blob/5c24d2e1c9c7966e9660a18d1a6e2037f914b773/lib/idp/constants.rb#L84-L100). While it wouldn't have helped for HTTP logging issues (something to look in to!), we do have [some guardrails to catching details from this PII constant in logging](https://github.com/18F/identity-idp/blob/5c24d2e1c9c7966e9660a18d1a6e2037f914b773/spec/support/fake_analytics.rb#L27) and I could imagine us doing something similar for HTTP request query parameters.